<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NormFilterDialog</class>
 <widget class="QDialog" name="NormFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>勾配ノルムフィルタ</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>横微分フィルタ(&amp;X):</string>
         </property>
         <property name="buddy">
          <cstring>textEditX</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTextEdit" name="textEditX">
         <property name="whatsThis">
          <string>フィルタ行列を入力します。</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>縦微分フィルタ(&amp;Y):</string>
         </property>
         <property name="buddy">
          <cstring>textEditY</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTextEdit" name="textEditY">
         <property name="whatsThis">
          <string>フィルタ行列を入力します。</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>例:</string>
       </property>
       <property name="buddy">
        <cstring>listWidgetFilter</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidgetFilter"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textEditX</tabstop>
  <tabstop>textEditY</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>listWidgetFilter</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NormFilterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NormFilterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidgetFilter</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>NormFilterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>640</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidgetFilter</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>NormFilterDialog</receiver>
   <slot>setExampleFilter(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>637</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setExampleFilter(QListWidgetItem*)</slot>
 </slots>
</ui>
